src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -11,135 +11,141 @@ namespace System.Text.RegularExpressions
     public partial class Capture
     {
         internal Capture() { }
-        public int Index { get { return default(int); } }
-        public int Length { get { return default(int); } }
-        public string Value { get { return default(string); } }
-        public override string ToString() { return default(string); }
+        public int Index { get { throw null; } }
```

Comment:
It is our new approach but we don't really have a good tool to run across all of them in the repo to fix this. As people are updating the references they can run genapi again and it will output this format. 

Actually I guess if people use the built reference assembly as input into genapi that would be a good way to normalize the ref cs files.